docs/reference/datatiers.asciidoc

```diff
@@ -10,8 +10,8 @@ typically share the same hardware profile:
 and hold your most recent, most-frequently-accessed data.
 * <<warm-tier, Warm tier>> nodes hold time series data that is accessed less-frequently
 and rarely needs to be updated.
-* <<cold-tier, Cold tier>> nodes hold time series data that is accessed infrequently and not normally updated.
-* <<frozen-tier, Frozen tier>> nodes hold time series data that is accessed rarely and never updated, kept in searchable snapshots.
+* <<cold-tier, Cold tier>> nodes hold time series data that is accessed infrequently and not normally updated. <<fully-mounted,Fully mounted indices>> of <<ilm-searchable-snapshot,{search-snaps}>> may be kept on the cold tier, eliminating the need for replicas with approximately 50% storage reduction compared to the warm tier.
```

Comment:
Two suggestions:

- I'd point out that the storage savings is in disk space.
- I'd point out that you can use regular indices on the cold tier if wanted.

```suggestion
* <<cold-tier,Cold tier>> nodes hold time series data that is accessed
infrequently and not normally updated. To save space, you can keep
<<fully-mounted,fully mounted indices>> of
<<ilm-searchable-snapshot,{search-snaps}>> on the cold tier. These fully mounted
indices eliminate the need for replicas, reducing required disk space by
approximately 50% compared to the regular indices.
+
Alternatively, you can the cold tier to store regular indices with replicas instead
of using {search-snaps}. This lets you store older data on less expensive hardware
but doesn't reduce required disk space compared to the warm tier.
```

Reply:
🤦 , I somehow missed this suggestion while I was working my way through the above.  I have added this commit in https://github.com/elastic/elasticsearch/pull/84155#discussion_r813521447 and in the main body.
 
To avoid duplication, I'd prefer to simplify the cold tier description in the summary at the top, and put the detailed description in the content, but I felt all of these points are important though, um...

Additionally, I am reading https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html and felt that some of the descriptions around cold or frozen data **nodes** can be updated accordingly or linked to content on this page.  Can you assist with that please?